test/modules/aws-elb-integration.ts

```diff
@@ -198,122 +167,3 @@ describe('ELB Integration Testing', () => {
     .remove(dbAlias, 'not-needed')
     .then(...finish(done)));
 });
-
-describe('ELB Integration Testing SP', () => {
```

Comment:
I think there is a few `SELECT` checks in this suite that are not in the one above?